tests/integration/targets/ipam_subnet/tasks/main.yml

```diff
@@ -266,12 +266,12 @@
           - subnet_info.objects | length == 1
           - subnet_info.objects[0].ddns_generate_name == true
 
-    - name: "Create a Subnet with ddns_send_updates set to false"
+    - name: "Create a Subnet with ddns_send_updates set to true"
```

Comment:
Its generally a good idea to test the attribute value other default values , hence in this case `false` would be a better input

Reply:
done